app/controllers/country_controller.rb

```diff
@@ -1,14 +1,30 @@
 class CountryController < ApplicationController
 
-  layout 'country'
+  layout 'application_react'
+  before_action :set_country, only: [:show]
 
-  def index
-    @desc = 'Data about forest change, tenure, forest related employment and land use'
+  def show
+    @title = @country["name"]
+    @widget = params[:widget]
```

Comment:
if you are not using a variable on the view you don't need to make it a instance variable `@variable` you can use just `variable`. Or in this case you can just use `params[:widget]`

Reply:
Ok will remove! Thanks for the review.

Reply:
Fixed :)